component/alertmanager.jsonnet

```diff
@@ -0,0 +1,101 @@
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.rancher_monitoring;
+local namespace = params.namespace;
+local instance = params.alertmanagerInstance;
+local name = 'alertmanager-' + instance;
+
+local labels = {
+  'app.kubernetes.io/name': 'alertmanager',
+  'app.kubernetes.io/instance': instance,
+  'app.kubernetes.io/managed-by': 'syn',
+};
+
+local matchLabels = {
+  app: 'alertmanager',
+  alertmanager: instance,
+};
+
+local sa = kube.ServiceAccount(name) {
+  metadata+: {
+    namespace: namespace,
+    labels+: labels,
+  },
+};
+
+local alertmanager = {
```

Comment:
Is there a specific reason to no use the alertmanager from `kube-prometheus`?

Reply:
For the initial open-source version I want to keep the resulting manifest diffs as small as possible to allow migrating to the open-source component with minimal effort.

We can switch to alertmanager from `kube-prometheus` in a follow-up issue/PR at any time if we think it's beneficial.